help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -47,8 +47,10 @@ Standards and Recommendations Section, Layout Control Specifications:
 <p>Note: <strong>C/MRI</strong> and <strong>CMRI</strong> (without the slash)
 are used interchangeably.  The C/MRI communication protocol is referred to as
 <strong>CMRInet</strong>.</p>
-<p style="margin-left:0em">The documentation below is divided into sections;
-for easy access click an item in this list:</p>
+<ul>
```

Comment:
HTML required that a UL element only contain LI elements. Some browsers are more generous than the standard, but not all are. What are you trying to achieve here?

Reply:
Corrected lines in error.  Now using Netbeans with html validation.